source/agora/utils/TxBuilder.d

```diff
@@ -508,125 +645,168 @@ unittest
     immutable Number = GenesisBlock.payments.front.outputs.length;
     assert(Number == 8);
 
-    const tx = TxBuilder(GenesisBlock.payments.front)
+    auto fee_rate = Amount(700);
+    const tx = TxBuilder(GenesisBlock.payments.front).feeRate(fee_rate)
```

Comment:
Isn't this the default ?

Reply:
Yes. We can also take the value from `Config`.

Reply:
Yes I have defaulted in `TxBuilder` constructor to use `ConsensusConfig.min_fee`